### PR TITLE
MKM 6 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/polygraph_orb.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Name:Polygraph Orb
+ManaCost:4 B
+Types:Artifact
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigDig | TriggerDescription$ When CARDNAME enters the battlefield, look at the top four cards of your library. Put two of them into your hand and the rest into your graveyard. You lose 2 life.
+SVar:TrigDig:DB$ Dig | DigNum$ 4 | DestinationZone2$ Graveyard | NoReveal$ True | ChangeNum$ 2 | SubAbility$ DBLoseLife
+SVar:DBLoseLife:DB$ LoseLife | LifeAmount$ 2
+A:AB$ Repeat | RepeatSubAbility$ RepeatTorment | RepeatPlayers$ Opponent | Cost$ 2 T CollectEvidence<3> | SpellDescription$ Each opponent loses 3 life unless they discard a card or sacrifice a creature. (To collect evidence 3, exile cards with total mana value 3 or greater from your graveyard.)
```
Comment (Simisays) [inline forge-gui/res/cardsfolder/upcoming/polygraph_orb.txt:7]: AI was stuck in an infinite loop, don't know how

File: forge-gui/res/cardsfolder/upcoming/perimeter_enforcer.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
+Name:Perimeter Enforcer
+ManaCost:1 W
+Types:Creature Human Detective
+PT:1/1
+K:Flying
+K:Lifelink
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Detective.YouCtrl | Execute$ TrigPump | TriggerDescription$ Whenever another Detective enters the battlefield under your control and whenever a Detective you control is turned face up, CARDNAME gets +1/+1 until end of turn.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/perimeter_enforcer.txt:7]: wrong valid

File: forge-gui/res/cardsfolder/upcoming/presumed_dead.txt [hunk 3]
```diff
@@ -0,0 +1,9 @@
+Name:Presumed Dead
+ManaCost:1 B
+Types:Instant
+A:SP$ Pump | ValidTgts$ Creature | NumAtt$ +2  | SubAbility$ DBAnimate | SpellDescription$ Until end of turn, target creature gets +2/+0 and gains "When this creature dies, return it to the battlefield under its owner's control and suspect it." (A suspected creature has menace and can't block.)
+SVar:DBAnimate:DB$ Animate | Triggers$ AbnormalEnduranceChangeZone | Defined$ ParentTarget
+SVar:AbnormalEnduranceChangeZone:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Card.Self | Execute$ AbnormalEnduranceTrigChangeZone | TriggerController$ TriggeredCardController | TriggerDescription$ When this creature dies, return it to the battlefield tapped under its owner's control and suspect it. (A suspected creature has menace and can't block.)
+SVar:AbnormalEnduranceTrigChangeZone:DB$ ChangeZone | Origin$ Graveyard | Destination$ Battlefield | Tapped$ True | Defined$ TriggeredNewCardLKICopy | SubAbility$ DBSuspect
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/presumed_dead.txt:7]: why tapped?`

File: forge-gui/res/cardsfolder/upcoming/polygraph_orb.txt (hunk 1 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/polygraph_orb.txt:7]: you used wrong Effect

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/polygraph_orb.txt]: but the real question is why are you even using Repeat when you're just iterating over them again below anyway?

**Review by tool4ever — APPROVED** [review]

**Pull request merged**